cmd/thanos/query.go

```diff
@@ -70,6 +70,13 @@ const (
 	promqlEngineThanos     promqlEngineType = "thanos"
 )
 
+type queryConnMetricLabel string
```

Comment:
Aditi's suggestion: Move it to endpointSet file so we can reference all following strings as `externalLabels`.